pages/deployment/[uuid].tsx

```diff
@@ -39,7 +39,19 @@ import { postEndpoint } from '../../data/api'
 
 const ComplianceFlow = dynamic(() => import('../../src/ComplianceFlow'))
 
-type TabOptions = 'overview' | 'compliance' | 'build'
+type TabOptions = 'overview' | 'compliance' | 'build' | 'settings'
```

Comment:
Instead of having this type and type guard below (`isTabOption`) defined in both files, could we just export from one and use in both?